code/datums/components/defiber.dm

```diff
@@ -0,0 +1,230 @@
+/datum/component/defiber
```

Comment:
defibrillator?
defiber is both a typo and unclear

Reply:
I felt it was confusing between defiber item and defiber component